static/docs/install/pre-release.md

```diff
@@ -1,29 +1,27 @@
 # Install Pre-release Version
 
-If you would like to install the latest version of DVC, you can do the
-following:
+If you want to test the latest stable version of DVC, ahead of official
+releases, you can install it from our code repository GitHub.
 
 > It's highly recommended using
 > [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments)
 > or
 > [pipx](https://packaging.python.org/guides/installing-stand-alone-command-line-tools/)
-> for Python 3.6+ to keep your system environment safe.
+> (on Python 3.6+) to encapsulate your local environment.
 
 ```dvc
+$ virtualenv -p python3 .env
```

Comment:
may be we should exclude those since this instruction is not special comparing to the regular one with pip

May be we can add a separate paragraph: "If you want to keep your previous version installed with pip, don't forget ..."

Reply:
Removed.